docroot/themes/humsci/humsci_basic/src/js/main-nav/menu-toggle.js

```diff
@@ -0,0 +1,56 @@
+const menuToggle = document.querySelector('.su-main-nav__toggle');
+const mainMenu = document.querySelector('.su-main-nav__menu-lv1');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+let isHidden;
+let wasDesktopSize;
+
+const collapseNav = () => {
+  menuToggle.setAttribute('aria-expanded', false);
+  mainMenu.setAttribute('aria-hidden', true);
+  menuToggle.innerHTML = "Menu";
+  isHidden = true;
+}
+
+const showNav = () => {
+  menuToggle.setAttribute('aria-expanded', true);
+  mainMenu.setAttribute('aria-hidden', false);
+  menuToggle.innerHTML = "Close";
+  isHidden = false;
+}
+
+if (menuToggle) {
+  // If below a certain breakpoint, hide the nav on page load
```

Comment:
this is more like if there is JS on the page, collapse the nav. we should set aria-expanded to true on the button trigger in this template code `docroot/themes/humsci/humsci_basic/templates/menus/menu--main.html.twig` by default on page load, and then this JS can come in if there is JS enabled on the page